index.js

```diff
@@ -0,0 +1,115 @@
+    artistData('Tego Calderon', ["Pa' Que Retozen", 'Metele Sazon', 'Dando Break'], 'A reggaeton pioneer known for his distinctive sound and socially conscious lyrics.', 'https://yt3.googleusercontent.com/kLvlPYWMrnvQVW6ObEtgjL5m5aZwBXaXTeGX6cxIOIV12iobB-kdfouBoXqIfsTH1tXcH-0mXg=s900-c-k-c0x00ffffff-no-rj'),
+    artistData('Carol G', ['Bichota', 'Tusa (with Nicki Minaj)', 'Ay, DiOs Mío!'], 'A rising reggaeton star celebrated for her dynamic vocals and empowering presence.', 'https://remezcla.com/wp-content/uploads/2023/07/header-GettyImages-1529560337-1424x1068.jpg'),
+    artistData('Bad Bunny', ['Dákiti (with Jhay Cortez)', 'Vete', 'Mía (with Drake)'], "A genre-defying phenomenon who's reshaped Latin music with his genre-blending creativity and chart-topping hits.", 'https://yt3.googleusercontent.com/7tCfeCWH4arhsTM-4Rz4IxWieQbegzibeXlG-kbytAujdk5dr2K0gBb8NG0Cvk6lB1dPkjyd=s900-c-k-c0x00ffffff-no-rj'),
+  ],
+};
+
+// Dynamically add featured artists
+const artistCard = (artistData, index) => {
+  const {
+    fullName, songs, blurb, imgUrl,
+  } = artistData;
+  const songsString = songs.reduce((acc, song, i) => {
+    if (i === songs.length - 1) {
+      return `${acc + song} `;
+    }
+    return `${acc + song}, `;
+  }, '');
+
+  return `
+    <div class="card card--${index}">
+      <figure class="portrait">
+        <svg viewBox="0 0 100 100" class="checkers">
+          <use xlink:href="sprite.svg#checkers"></use>
+        </svg>
+        <img src="${imgUrl}" alt="${fullName}">
+      </figure>
+      <div class="profile">
+        <h3>${fullName}</h3>
+        <h5>Hits: ${songsString}</h5>
+        <p class="description">${blurb}</p>
+      </div>
+    </div>
+  `;
+};
+const featuredCards = grab('.cards');
+const { artists } = db;
+const featuredArtists = artists.reduce((acc, artist, i) => acc + artistCard(artist, i), '');
+render(featuredCards, featuredArtists);
+
+// Show only 2 artists in mobile
+const artistCards = Object.values(grab('.cards').children);
+const visibleCards = [0, 1];
+const renderMobileCards = () => {
+  artistCards.forEach((card, i) => {
+    if (visibleCards.includes(i)) {
+      card.classList.remove('hide');
+    } else {
+      card.classList.add('hide');
+    }
+  });
+
+  if (visibleCards[0] === 4) {
+    visibleCards[0] = 0;
+    visibleCards[1] = 1;
+  } else {
+    visibleCards[0] += 2;
+    visibleCards[1] += 2;
+  }
+};
+
+renderMobileCards();
+
+// activate button to cicle through featured artists on mobile version
+const moreArtists = grab('.more');
+onClick(moreArtists, renderMobileCards);
+
+// Toggle mobile menu
+const menuBurger = grab('.menu-burger');
+const header = grab('header');
+const toggleMenu = () => {
+  const { className } = header;
+  if (className === 'header') {
+    header.className = 'openNav';
+  } else {
+    header.className = 'header';
+  }
+};
+onClick(menuBurger, toggleMenu);
```

Comment:
- [ ] If you notice, most of the websites show the cross icon(x) when the mobile menu is shown. I suggest that you consider doing so to ensure that the UI of your webpage is improved
- [ ] [OPTIONAL] Again on the same note as the mobile menu, I suggest that when you click the About link, you consider navigating to the About page, and the mobile menu is closed upon the click ☺️. 

Your design 😎 
![image](https://github.com/Drod86/reggaeducaton/assets/99394374/68c1729f-b877-4f84-acbc-08ff83807dd6)

Reply:
Hello @AlphaNtihinduka. Thank you for your valuable feedback. I implemented all of your changes as mentioned above.